src/api/repos.rs

```diff
@@ -66,6 +66,22 @@ impl<'octo> RepoHandler<'octo> {
         self.crab.get(url, None::<&()>).await
     }
 
+
+    /// Fetches a repository's metrics.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// let repo = octocrab::instance()
+    ///     .repos("owner", "repo")
+    ///     .get_metrics()
+    ///     .await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub async fn get_metrics(&self) -> Result<models::RepositoryMetrics> {
```

Comment:
Should be named `get_community_profile_metrics` to match octokit. https://octokit.github.io/rest.js/v18#repos-get-community-profile-metrics
```suggestion
    pub async fn get_community_profile_metrics(&self) -> Result<models::RepositoryMetrics> {
```